Library/Formula/duck.rb

```diff
@@ -1,16 +1,9 @@
 class Duck < Formula
   homepage "https://duck.sh/"
-  url "https://dist.duck.sh/duck-src-4.6.2.16366.tar.gz"
-  sha1 "135f7ca7315fd478607e590eaa2df43418d85e0d"
+  url "https://dist.duck.sh/duck-src-4.6.2.16480.tar.gz"
+  sha1 "6e38a90c97bbe7d577791a4fbdd07872e8457101"
   head "https://svn.cyberduck.io/trunk/"
 
-  bottle do
-    cellar :any
-    sha1 "05353941783e34f1931b0cdd78a89f245d8daf33" => :yosemite
-    sha1 "a2120b7b4b7ae15f9b941c24bb9d4f7c36392e74" => :mavericks
-    sha1 "fb6e8e20d702cf3d9808e758328cd7ebc8830283" => :mountain_lion
-  end
```

Comment:
They will be but it helps the detection algorithm if they are still present in the formula. Thanks!